.gitignore

```diff
@@ -63,3 +63,4 @@ sources
 var/*
 venv
 virtualenv.py
+requirements/dev.txt
```

Comment:
ignoring the lockfiles contradicts the idea of having pins for reproducible envs.
```suggestion
```

Reply:
Done.